src/adapters/poi/idunn_poi.js

```diff
@@ -103,6 +103,10 @@ export default class IdunnPoi extends Poi {
     } catch (err) {
       if (err === 404) {
         return;
+      } else if (err === 0) { // When the OPTIONS request is rejected, the error is 0 and not 405
```

Comment:
`0` isn't a valid HTTP status code, is it a custom one from Idunn API? And is it guaranteed this will be used only for this error case? Otherwise it's a bit dangerous to resend the request in this `catch`, you may enter an infinite loop of requests…

Reply:
Oh good point, I need to add a check to confirm that we should resend it.

Also, not sure why 0 is returned (my guess is that it comes from the fact that it is the OPTIONS request which is failing) but since we're in the catch block, I assume it's fine?